common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -487,6 +487,8 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+
+  // Owner: Joseph Smith
   val AmpOphanTrackingIncludesCredentials = Switch(
```

Comment:
@SiAdcock  Thanks for that - I think I messed up the conflict resolution when merging from master. It should be gone now.